drivers/spi/Kconfig.nrfx

```diff
@@ -14,6 +14,19 @@ menuconfig SPI_NRFX
 
 if SPI_NRFX
 
+config SOC_NRF52832_PAN_58_OVERRIDE
+	depends on SOC_NRF52832
+	bool "Allow enabling the SPIM driver despite PAN 58"
+	help
+	  Allow enabling the nRF SPI Master with EasyDMA, despite
+	  Product Anomaly Notice 58 (SPIM: An additional byte is
+	  clocked out when RXD.MAXCNT = 1). Without this
+	  override, the SPI Master is only available without
+	  EasyDMA. Note that the 'SPIM' and 'SPIS' drivers
+	  use EasyDMA, while the 'SPI' driver does not. Use
+	  this option ONLY if you are certain that transactions
+	  with RXD.MAXCNT = 1 will not be executed.
```

Comment:
Now it is a bit imprecise and therefore it restricts too much. There are two conditions for the anomaly to appear: RXD.MAXCNT = 1 and TXD.MAXCNT <= 1. Thus, I suggest adding "when TXD.MAXCNT <= 1" in the above statement.

Reply:
Good idea - done.

I also made all the verbiage consistent everywhere that this appears.